pinc/faq.inc

```diff
@@ -204,14 +206,11 @@ function get_external_faq($faq, $langcode)
     return null;
 }
 
-function maybe_redirect_to_external_faq($langcode = null)
```

Comment:
We can drop this function entirely now.

Reply:
Removed this function in the updated PR.